common/commands/errors.cc

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2019 IOTA Stiftung
+ * https://github.com/iotaledger/rpchub
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "common/commands/errors.h"
+
+namespace common {
+
+namespace cmd {
+
+std::map<Error, std::string> errorToStringMapCreate() {
+  std::map<Error, std::string> map;
+  map[OK] = "OK";
+  map[MISSING_ARGUMENT] = "MISSING_ARGUMENT";
+  map[WRONG_ARGUMENT_NAME] = "WRONG_ARGUMENT_NAME";
```

Comment:
Why are some uppercase + underscore and some just usual string ?

Reply:
I guess i was drunk